modules/ROOT/pages/message-logging-policy.adoc

```diff
@@ -3,53 +3,55 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This policy logs a custom message with the information available between policies, proxy, or backend at any point of the execution. This policy works with both log4j and log4j2.
-
-== Prerequisites
-
-You need to have the Anypoint Platform organization administrator role or have permission to create or manage APIs.
-
-== About Logging Policy Parameters
-
-The Message Logging Policy consists of these configuration parameters:
-
-image::apim-message-logging-policy-ui.png[Message Logging Policy UI]
-
-[%header%autowidth.spread]
+[width="100%", cols="5,15"]
 |===
-|Parameter |Description
-|Message |A DataWeave expression that's transformed into a message to be logged.
-
-Note: If the payload is used as part of the message log, do not configure the Listener as non-repeatable (by default this setting is off).
-|Conditional |The condition expressed as a DataWeave expression that returns true or false that needs to be satisfied to log the message. This field is optional.
-|Category |The category that the log has. This parameter is optional. If no category is set, the default is the `org.mule.runtime.logging.policy-<policy id>` value.
-|Level |Defines if the message is logged according to the configuration in your log4j2 file.
-|Before Calling API |Set to cause your logging policy to be applied before the API is called.
-|After Calling API |Set to cause your logging policy to be applied after the API is called.
+>s| Policy Name | Message Logging
+>s|Summary      | Logs messages user-password defined on LDAP
```

Comment:
"Logs custom messages. Content can be composed using information from the incoming request."
... Or similar.

Reply:
Done! Thanks!